front/app/containers/EventsShowPage/EventTopBarMobile.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
```

Comment:
Any reason this one isn't in `EventsShowPage/components`?

Reply:
Ah good point! Nope, I'll move it :)